New package: PolymerArchitecture v0.3.8

JuliaRegistrator opened the pull request:
- Registering package: PolymerArchitecture
- Repository: https://github.com/liuyxpp/PolymerArchitecture.jl
- Created by: @liuyxpp
- Version: v0.3.8
- Commit: ad77d8e739c5746b40a67754536fdc34dc8abd94
- Reviewed by: @liuyxpp
- Reference: https://github.com/liuyxpp/PolymerArchitecture.jl/commit/ad77d8e739c5746b40a67754536fdc34dc8abd94#commitcomment-173158113
- Description: A Julia package that provides a graph representation of the polymer architecture.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Add docs
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd663e673070bb3a9a09fedccdaaf54ea2770c6039e14a8e657a4d89c2116faca1c923c11908e48c26be292fd60dd22dc6031b550849765eb04dffed0c80c951eacb998e85e6a11f6b3b3ff9a5a337e450d38511e4e9380dba418b54267b0cb19e5d5fe919d7a05c925adb012ae97aed4c032050edb7bbcd52a1a04e76e93d87ee3d46d030bcb65a65c8fcbf7f450df63146b70ca758eaf096ce0e80ab6133f008f249dae8024a59da113be989a35995a61 -->

goerz commented:
Thank you for submitting your package! I would ask that you add add a little bit more documentation before registering than

> Documentation is not available yet. However, one can consult the test codes reside in the test folder to learn more.

[At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a  small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though.

goerz commented:
You just comment `@JuliaRegistrator register` on the new commit to update this existing PR. Do not change the version number (that would create a new PR)

[noblock]

JuliaRegistrator committed:
New package: PolymerArchitecture v0.3.8

UUID: 9f77f41a-3be8-4884-a3be-1fccfab54d3c
Repo: https://github.com/liuyxpp/PolymerArchitecture.jl.git
Tree: 771c267bf21171172fd49b692a8cab2d34499d22

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
